draft-ietf-quic-http.md

```diff
@@ -2174,8 +2174,7 @@ NO_ERROR (0x0):
 : HTTP_NO_ERROR in {{http-error-codes}}.
 
 PROTOCOL_ERROR (0x1):
-: No single mapping.  See new HTTP_MALFORMED_FRAME error codes defined in
-  {{http-error-codes}}.
+: HTTP_GENERAL_PROTOCOL ERROR in {{http-error-codes}}.
```

Comment:
Isn't it still the case that we have multiple error codes that map to HTTP_GENERAL_PROTOCOL error in HTTP/2?

Or are we going to merge all the connection-level error codes that do not map to an error code in HTTP/2 (e.g., WRONG_STREAM, UNEXPECTED_FRAME, CLOSED_CRITICAL_STREAM) to HTTP_GENERAL_PROTOCOL?

Reply:
I don't follow sorry. This section states:

> QUIC has the same concepts of “stream” and “connection” errors that HTTP/2 provides. However, there is no direct portability of HTTP/2 error codes.

The change is intended to address how the HTTP/2 error code PROTOCOL_ERROR for HTTP/3 non-connection errors (usually frame errors) are expressed.

How would you suggest improving this?

Reply:
Maybe I should have stated that the removal of "No single mapping." is concerning, because IIUC we do not have one-to-one mapping between PROTOCOL_ERROR (of HTTP/2) and HTTP_GENEAL_PROTOCOL error (of HTTP/3).

What I had in mind is something like "Split into a few error codes, including HTTP_GENERAL_PROTOCOL, HTTP_WRONG_STREAM, HTTP_UNEXPECTED_FRAME, HTTP_CLOSED_CRITICAL_STREAM."

Reply:
Thanks, I agree and would like to adopt that text. The annoying thing is that it add more references to error codes I'm looking to consolidate but I can try to update the text to match the state of the draft as the PRs get merged.